.github/workflows/docker.yml

```diff
@@ -61,7 +61,7 @@ jobs:
       # On master, head_ref is empty, so we use the SHA of the commit, this means
       # commits to master will not be cancelled, which is important to ensure
       # that every commit to master is full tested and deployed.
-      group: docker-${{ matrix.docker-image }}-${{ github.head_ref || github.sha }}
+      group: docker-${{ matrix.docker-image }}-${{ github.ref }}-${{ github.head_ref || github.sha }}
```

Comment:
Since github.ref and github.head_ref both refer to a branch, we will cancel different commits on the same branch (for pull requests)? Does a scheduled daily build conflict with tags?

Reply:
I think it would, but the likelihood of overlap is pretty low. Still, more reason for GitHub to all conditional concurrency groups.